#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#
Header header
float32 yaw
float32 pitch
float32 roll
float32 yawRot
float32 pitchRot
float32 rollRot